packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,58 +12,58 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--ContentMarginBlockEnd);
   margin-inline: 0;
-  color: $text-heading;
-  font-family: $body-font-family;
-  font-size: $base-font-size;
+  color: var(--TextColor);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
   font-weight: 600;
 }
 
 .level1 {
-  font-size: $size-heading-1;
+  font-size: var(--Size1FontSize);
 }
 
 .level2 {
-  font-size: $size-heading-2;
+  font-size: var(--Size2FontSize);
 }
 
 .level3 {
-  font-size: $size-heading-3;
+  font-size: var(--Size3FontSize);
 }
 
 .level4 {
-  font-size: $size-heading-4;
+  font-size: var(--Size4FontSize);
 }
 
 .level5 {
-  font-size: $size-heading-5;
+  font-size: var(--Size5FontSize);
 }
 
 .level6 {
-  font-size: $size-heading-6;
+  font-size: var(--Size6FontSize);
 }
 
 .level1,
 .level2,
 .level3 {
-  line-height: $heading-line-height;
+  line-height: var(--LineHeightFontHeading);
 }
 
 .level4,
 .level5,
 .level6 {
-  line-height: $base-line-height;
+  line-height: var(--FontLineHeight);
 }
 
 .noEndMargin {
   margin-block-end: 0;
 }
 
 .baseLineHeight {
-  line-height: $base-line-height;
+  line-height: var(--LineHeight);
 }
 
 .headingLineHeight {
-  line-height: $heading-line-height;
+  line-height: var(--AltLineHeight);
 }
```

Comment:
I had forgotten about these exception styles. We had originally planned to revisit Heading so that we could remove these props and styles and allow props to be passed through to the Box instance it composes so Heading didn't need to own these styles. [Very much related to the RFC here too.](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2469433541/Odyssey+Theming+Duplicate+CSS+Architecture+RFC)

for now I think I would favor names that follow the public prop based API, so rather than `Alt` use `Heading` and simple add that token into your eslint escaped comment block

Reply:
Im confused by this, If im a downstream developer and I see "Alt" and "Heading" how do I know what exactly those properties will do? It feels unintuitive.

Reply:
my point above is really that the names we choose for the tokens should align with the public API that already exists in props! we don't have a `lineHeight="alt"` prop but we do have `lineHeight="heading"` yes?